low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R2.yaml

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
please take latest yaml files from @shriya-deshmukh as i have given some review comments in her PR

Reply:
Updated conf files as per comments